### DISCUSSION TITLE
updating `xz` to `5.6.2`

### DISCUSSION THREAD

**Discussion by johnalanwoods** [Homebrew/discussions#5437] [Everyday usage] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Should we not update `xz` to [5.6.2](https://github.com/tukaani-project/xz/releases/tag/v5.6.2)? Lasse released it two weeks ago, I think it's likely safer than Jia's commit of `5.4.6`?

**Comment by SMillerDev** [comment, marked as answer]
https://github.com/Homebrew/homebrew-core/pull/173167, feel free to revive it

**Reply by johnalanwoods** [reply]
sorry for the stupid question but not used to the flows, why was this one closed without merge?

**Reply by SMillerDev** [reply]
Because nobody was fixing it.
